### DISCUSSION TITLE
Listen also on port 8448 for one address?

### DISCUSSION THREAD

**Discussion by smooth-baking-powder** [tobychui/zoraxy#438] [Q&A]
Hi,

can i listen the reverse proxy on 8448 also? 443 already work. i added 8448 on container and now i need to add the subdomain with port 8448.

Thanks

**Comment by tobychui** [comment]
Zoraxy currently only support one inbound (downstream to Zoraxy) listening port. Though there are nothing to stop you for using tcp proxy to forward request from an alternative port like 8448 to your primary port like 443.
